spec/requests/versions_spec.rb

```diff
@@ -17,7 +18,7 @@
 
     context 'reverts the version of the case' do
```

Comment:
Can we add a spec to handle the failure case of the controller, when the `version.reify` returns nil or a `StandardError` is returned?

Reply:
Sure thing!

Reply:
I was thinking of something like:
```
context 'when version.reify fails' do
      let(:version_mock) { double("version") }
      before do
        allow(PaperTrail::Version).to receive(find_by_id) { version_mock }
        post "/cases/#{this_case.id}/versions/#{version_id}/revert",
           params: {},
           headers: {
             "HTTP_REFERER": '/'
           }
      end

      it 'redirects to the previous page' do
        expect(response).to redirect_to("/cases/#{this_case.slug}")
      end

      it 'does not create a new version' do 
        # TODO: compare versions before & after method call
      end
    end
```

but that's not central to upgrading, so that can be worked on separately.